packages/backend/src/tokens/handshake.ts

```diff
@@ -0,0 +1,69 @@
+import { TokenVerificationError, TokenVerificationErrorAction, TokenVerificationErrorReason } from './errors';
+import { decodeJwt, hasValidSignature, type VerifyJwtOptions } from './jwt';
+import { assertHeaderAlgorithm, assertHeaderType } from './jwt/assertions';
+import { loadClerkJWKFromLocal, loadClerkJWKFromRemote } from './keys';
+import type { VerifyTokenOptions } from './verify';
+
+export async function verifyHandshakeJwt(token: string, { key }: VerifyJwtOptions): Promise<{ handshake: string[] }> {
```

Comment:
@BRKalow Do we need to export this? If not let's keep it local to avoid confusion between verifyJWT and verifyToken methods.

Reply:
Sure don't! Thanks, good catch.